addons/wp/functions/fnc_taskAssault.sqf

```diff
@@ -78,11 +83,10 @@ private _fnc_reached = {
 // Function ~ restore
 private _fnc_restore = {
     params ["_unit","_handle"];
-    if (_handle != (_group getVariable ["lambs_danger_forcedMovement",-1])) exitWith {};
+    if (_handle != (_group getVariable [QGVAR(forcedMovement),-1])) exitWith {};
```

Comment:
This will translate to `lambs_wp_forcedMovement`, is this intended?

Reply:
It is.  I thought about it, and the forced movement variable is only queried in the WP.  Do you think it should be linked to lambs_danger instead?

Reply:
If it is only used in the  WP module, it is fine as that, but if it is also used in the danger component, then it should be using the danger module prefix imo

Reply:
Agreed. Will fix.